src/components/MediaTile/MediaTile.tsx

```diff
@@ -100,49 +29,87 @@ export type MediaTileProps = MediaTileBaseProps &
   );
 
 const MediaTile: React.FC<MediaTileProps> = props => {
-  const { loading, onDelete, onEdit, editHref, media, disableOverlay = false } = props;
-  const classes = useStyles(props);
+  const {
+    loading,
+    onDelete,
+    onEdit,
+    editHref,
+    media,
+    disableOverlay = false,
+    draggable = false,
+  } = props;
   const parsedMediaOembedData = media?.oembedData ? JSON.parse(media.oembedData) : null;
   const mediaUrl = parsedMediaOembedData?.thumbnail_url || media.url;
 
+  const [isWrapperHover, setIsWrapperHover] = useState(false);
+
   return (
-    <div className={classes.mediaContainer} data-test-id="product-image">
-      <div
-        className={clsx(classes.mediaOverlay, {
-          [classes.mediaOverlayShadow]: loading,
-          [classes.disableOverlay]: disableOverlay,
-        })}
+    <Box
+      data-test-id="product-image"
+      backgroundColor="default1"
+      borderRadius={4}
+      __width="150px"
+      __height="150px"
+      padding={1}
+      position="relative"
+      borderStyle="solid"
+      borderWidth={1}
+      borderColor="default1"
+      onMouseEnter={() => !disableOverlay && setIsWrapperHover(true)}
+      onMouseLeave={() => !disableOverlay && setIsWrapperHover(false)}
+    >
+      <Box
+        backgroundColor="default1"
+        __opacity={0.8}
+        borderRadius={4}
+        position="absolute"
+        left={0}
+        top={0}
+        __width="148px"
+        __height="148px"
+        display={isWrapperHover ? "block" : "none"}
+        __cursor={draggable ? "move" : "default"}
+      />
+      <Box
+        className="show-on-hover"
```

Comment:
queston: do you still need this className?

Reply:
ah, forgot